src/main/app/case/from-api-format.ts

```diff
@@ -15,6 +15,28 @@ const fields = {
     englishOrWelsh:
       data.LanguagePreferenceWelsh === YesOrNo.Yes ? LanguagePreference.Welsh : LanguagePreference.English,
   }),
+  D8DerivedPetitionerHomeAddress: data => {
+    if (!data.D8DerivedPetitionerHomeAddress) {
```

Comment:
when does this happen?

Reply:
👍 Good spot, this shouldn't, I've removed it